notebooks/basics/multiple-systems.ipynb

```diff
@@ -528,38 +565,37 @@
     "\n",
     "This is the formula for the so-called *reduced* (or *marginal*) probabilistic state of $\\mathsf{X}$ alone.\n",
     "\n",
-    "This formula makes perfect sense at an intuitive level: something very strange would need to happen for it to be wrong.\n",
-    "If the formula was not correct, it would mean that the probabilities of obtaining different outcomes when $\\mathsf{X}$ is measured could somehow be influenced simply by whether or not $\\mathsf{Y}$ was also measured, irrespective of the outcome.\n",
-    "If $\\mathsf{Y}$ happened to be in a distant location, for instance, this would allow for superluminal signaling, which we reject based on our understanding of physics.\n",
+    "This formula makes perfect sense at an intuitive level; something very strange would need to happen for it to be wrong. It would mean that the probabilities for $\\mathsf{X}$ measurements are influenced simply by whether or not $\\mathsf{Y}$ is also measured, irrespective of the outcome on $\\mathsf{Y}$.\n",
+    "If $\\mathsf{Y}$ happened to be in a distant location, say, another galaxy, this would allow for superluminal signaling, which we reject based on our understanding of physics. Another way to understand this comes from an interpretation of probability as reflecting degree of belief about the state of the system. Since a measurement on $\\mathsf{Y}$ is taken to simply reveal a preexisting state, a different observer looking at $\\mathsf{X}$, unaware of the $\\mathsf{Y}$ measurement, should not have their probabilities changed.\n",
```

Comment:
"Since a measurement on $\\mathsf{Y}$ is taken to simply reveal a preexisting state..."
->
"Because a measurement on $\\mathsf{Y}$ simply reveals a preexisting state..."

[Also slightly worried that we're beating a dead horse. The idea is pretty simple to grasp.]